score-v1b1.json

```diff
@@ -338,13 +338,29 @@
     },
     "containerProbe": {
       "type": "object",
-      "required": [
-        "httpGet"
-      ],
+      "description": "The probe may be defined as either http, command execution, or both when the deployment runtime supports one and not the other.",
```

Comment:
Should the spec define which one takes precedence if both are supported - or should that be implementation specific?